code/modules/mob/living/carbon/xenomorph/castes/drone/abilities_drone.dm

```diff
@@ -111,14 +110,14 @@
 	var/remaining_health = round(target.maxHealth - (target.getBruteLoss() + target.getFireLoss()))
 	var/health_threshold = round(target.maxHealth / 10) // 10% of the target's maximum health
 	var/heal_multiplier = 1
-	if(essence_link_action.existing_link && target == essence_link_action.existing_link.link_target)
+	if(essence_link_action?.existing_link && target == essence_link_action?.existing_link.link_target)
```

Comment:
In that case you do not need the first check

Reply:
Done in next commit.